packages/drizzle-duckdb-wasm/src/dsn.ts

```diff
@@ -0,0 +1,115 @@
+import type { DBStorage } from '@proj-airi/duckdb-wasm'
+
+import { DuckDBAccessMode } from '@duckdb/duckdb-wasm'
+import { DBStorageType } from '@proj-airi/duckdb-wasm'
+
+export interface StructuredDSN {
+  protocol: 'duckdb-wasm:'
```

Comment:
```suggestion
  scheme: 'duckdb-wasm:'
```

Ref: https://developer.mozilla.org/en-US/docs/Web/URI/Reference/Schemes